intrahospital_api/management/commands/merge_patients_since.py

```diff
@@ -0,0 +1,38 @@
+import datetime
+import time
+import traceback
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+from opal.models import Patient
+from intrahospital_api import update_demographics, logger, constants
+from plugins.monitoring.models import Fact
+from elcid import utils
+
+UNEXPECTED_VOLUMES_THRESHOLD = 200000
```

Comment:
I was thinking more like 500?

Reply:
Sounds good, over the last month there have been 3 days were last_updated > 300, 1 instance where its been above 400 so I think 1000 is better than 500